enhancements/baremetal/enable-baremetal-on-aws.md

```diff
@@ -0,0 +1,152 @@
+---
+title: enable-baremetal-on-aws
+authors:
+  - "@elfosardo"
+reviewers:
+  - "@dtantsur"
+  - "@zaneb"
+approvers:
+  - "@dtantsur"
+  - "@zaneb"
+api-approvers:
+  - "@dtantsur"
+  - "@zaneb"
+creation-date: 2023-01-30
+last-updated: 2023-01-31
+tracking-link: 
+  - "https://issues.redhat.com/browse/METAL-300"
+status: implementable
+see-also:
+  - "/enhancements/baremetal/baremetal-provisioning-config.md"
+  - "/enhancements/baremetal/enable-baremetal-on-other-platforms.md"
+replaces:
+superseded-by:
+---
+
+# Enable baremetal on AWS to support centralized host management
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Baremetal Host API is available when deploying an OpenShift cluster with the baremetal platform (via the IPI or AI (Assisted Installer) workflow) or with the on-premise platforms Openstack, vSphere, and None.  
+Adding the possibility to deploy and manage bare metal hosts from clusters not in an on-premise infrastructure would be beneficial to customers.
+
+## Motivation
+
+Having the control plane on a pure cloud platform, allowing the deployment and management of bare metal workers, provides a true hybrid cloud solution.
+
+### Goals
+
+Support the centralized host management use case by enabling Baremetal Host API on AWS platform, in order to allow clusters deployment when running on the AWS platform.
+
+### Non-Goals
+
+* Allow Baremetal Host API to be fully enabled on other cloud platforms, such as GCP and Azure.
+* Automatically configure the connection between the AWS platform and the user's bare metal hosts.
+
```

Comment:
I assume making Machine API functional is also not a goal? Worth stating explicitly.